bin-example/generation/build.gradle

```diff
@@ -38,47 +54,63 @@ jar {
   }
 }
 
-sourceCompatibility = 1.7
-targetCompatibility = 1.7
+task startGenEnv(overwrite: true, type: Exec) {
+  executable = './../../generation-environment.sh'
+  args = [project.name]
+}
 
-repositories {
-    mavenLocal()
-    mavenCentral()
+task stopGenEnv(overwrite: true, type: Exec) {
+  executable = './../../generation-environment.sh'
+  args = ["stop", project.name]
 }
 
-configurations {
-    provided
-    compile.extendsFrom provided
+task removeGenEnv(overwrite: true, type: Exec) {
+  executable = './../../generation-environment.sh'
+  args = ["rm", project.name]
+}
+
```

Comment:
Need a way to clean up the output/ directory. Either with an existing task, or a new one